app/utils/hash-util.js

```diff
@@ -0,0 +1,4 @@
+export function setHash(pieceName) {
+  const hash = pieceName.replaceAll(" ", "-");
+  window.location.hash = hash;
```

Comment:
replacing spaces with dashes could lead to excessive dashes
`VR-PV-2024/7---ALLE-BESLISSINGEN` no idea if that is ok.

Any other symbols we need to replace (escape characters perhaps? & " ') If any are present. They might just get translated to symbols `%20`
need to check existing data for special cases.

Reply:
One thing I noticed here is that the downloaded file uses file name, not piece name.
Alvin was next to me when I was testing this. We should be using the piece name instead of the file name I think
Will discuss with Tom and probably make a ticket.
I think we do use piece name when downloading all documents from agenda, but in some places, we are offering the download with a different name than we see in the app (the former which cannot be changed)